src/util.js

```diff
@@ -41,6 +41,21 @@ Util.isIFrame = function() {
   }
 };
 
+// Get a unique, incrementing Id value for any object on the page.
+Util.getUniqueId = (function(prefix) {
+  var i = Math.floor(Math.random() * 999) + 100;
+  var pfx = prefix || '';
+  function inc(pfx) {
+    if (!pfx) {
+      pfx = '';
+    } else {
+      pfx += '-';
+    }
+    return pfx + i++;
+  }
+  return inc;
+})();
```

Comment:
the was that the first time the method runs, it does something different (get a random number) than on subsequent runs (increment the number). But that doesn't matter to the caller, so will change.